GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -23,63 +23,64 @@ private void InitializeComponent()
         {
             this.components = new System.ComponentModel.Container();
             this.UnstagedFileContext = new System.Windows.Forms.ContextMenuStrip(this.components);
+            this.stageToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
```

Comment:
What is the essence of your change? Cherry pick it, please, and keep the original sequence in the file.

Reply:
I didn't want to commit these changes, sorry...